### ISSUE TITLE
Relicensing `wasm-tools` and this repository's crates

### ISSUE THREAD

**Issue by alexcrichton** [bytecodealliance/wasm-tools#1637]
# Relicensing `wasm-tools`

Hello everyone! This issue is here to serve as a checklist for relicensing the `wasm-tools` repository and all crates that it contains.

* **Before**: `Apache-2.0 WITH LLVM-exception`
* **After**: `Apache-2.0 WITH LLVM-exception OR Apache-2.0 OR MIT`.

In other words this issue is adding two other licensing options to the crate, the Rust standard `Apache-2.0 OR MIT` option. Everyone who has contributed to this repository is cc'd below with a checkbox tracking their approval status.

If you approve of this license change it's requested you leave a comment along the lines of "I approve this license change" and your checkbox will be filled in by a maintainer.

## Why change licenses?

The original choice of `Apache-2.0 WITH LLVM-exception` for this repository matches the licensing decision of Wasmtime itself and was intended to be compatible with it. Some crates in this repository, however, would like to be integrated into upstream Rust itself. For example the `wasmparser` and `wasm-encoder` crates are already part of the Rust compiler. Currently the Rust compiler [has an exception][exception] which allows these crates despite not having the Rust-standard `Apache-2.0 OR MIT` license. The goal of this issue is to make integration in the upstream Rust compiler easier by ensuring that the license is fully compatible with Rust's existing license.

[exception]: https://github.com/rust-lang/rust/blob/d929a42a664c026167800801b26d734db925314f/src/tools/tidy/src/deps.rs#L97-L98

## Contributors

Contributors to `wasm-tools` are generated with `git shortlog --summary
--numbered --email` as of [28e96f680752c999f296e5f9d7d31a6be66112f5](https://github.com/bytecodealliance/wasm-tools/commit/28e96f680752c999f296e5f9d7d31a6be66112f5).

* [x] @abrown ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=abrown))
* [x] @adambratschikaye ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=adambratschikaye))
* [x] @alexcrichton ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=alexcrichton))
* [x] @andylizi ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=andylizi))
* [x] @azaslavsky ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=azaslavsky))
* [x] @bakkot ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=bakkot))
* [x] @bvisness ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=bvisness))
* [x] @cfallin ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=cfallin))
* [x] @darinmorrison ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=darinmorrison))
* [x] @dhil ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=dhil))
* [x] @dicej ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=dicej))
* [x] @elliottt ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=elliottt))
* [x] @eqrion ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=eqrion))
* [x] @fibonacci1729 ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=fibonacci1729))
* [x] @fitzgen ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=fitzgen))
* [x] @guybedford ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=guybedford))
* [x] @imikushin ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=imikushin))
* [x] @itsrainy ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=itsrainy))
* [x] @jacarte ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=Jacarte))
* [x] @jbms ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=jbms))
* [x] @jpages ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=jpages))
* [x] @julian-seward1 ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=julian-seward1))
* [x] @juntyr ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=juntyr))
* [x] @kateinoigakukun ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=kateinoigakukun))
* [x] @lann ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=lann))
* [x] @lars-t-hansen ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=lars-t-hansen))
* [x] @liamolucko ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=Liamolucko))
* [x] @maekawatoshiki ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=maekawatoshiki))
* [x] @mainrs ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=mainrs))
* [x] @mbebenita ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=mbebenita))
* [x] @mendyBerger ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=MendyBerger))
* [x] @mossaka ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=Mossaka))
* [x] @nagisa ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=nagisa))
* [ ] ~~@nlewycky~~ @syrusakbary ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=nlewycky))
* [x] @pchickey ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=pchickey))
* [x] @peterhuene ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=peterhuene))
* [x] @robbepop ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=Robbepop))
* [x] @rylev ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=rylev))
* [x] @silesmo ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=silesmo))
* [x] @sunfishcode ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=sunfishcode))
* [x] @takikawa ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=takikawa))
* [ ] @theGreatRambler ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=TheGreatRambler))
* [x] @yaronWittenstein ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=YaronWittenstein))
* [x] @ydnar ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=ydnar))
* [x] @yoshuawuyts ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=yoshuawuyts))
* [x] @yowl ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=yowl))
* [x] @yurydelendik ([commits](https://github.com/bytecodealliance/wasm-tools/commits?author=yurydelendik))

## Other contributors

As you can see, we have a long list of contributors to wasm-tools. However, it's not even the entire list! Lots of other people helped us by providing small tweaks to the code, fixing grammar issues in documentation, and similar contributions. While these kinds of contributions are extremely valuable, they aren't covered by copyrights, and thus we don't require sign-off by the authors of these contributions.

We do want to use this opportunity to list everyone who contributed to wasm-tools in whichever way though, so here's the list of people we don't need sign-off from, including links to their contributions:

* Adam S Levy [388ca0b315bb6e838d07f24fec164b60611d2236](https://github.com/bytecodealliance/wasm-tools/commit/388ca0b315bb6e838d07f24fec164b60611d2236)
* Aleksey Kladov [0209fc72bfcb8ce2f4d7df430e08555eb60af1c2](https://github.com/bytecodealliance/wasm-tools/commit/0209fc72bfcb8ce2f4d7df430e08555eb60af1c2) [ca97cf29dffed5b7c2cf1b84495fb3fad6f7a1e0](https://github.com/bytecodealliance/wasm-tools/commit/ca97cf29dffed5b7c2cf1b84495fb3fad6f7a1e0)
* Amila Welihinda [35ae6da77e65b931b709785f1fb5bd53fae2a7a8](https://github.com/bytecodealliance/wasm-tools/commit/35ae6da77e65b931b709785f1fb5bd53fae2a7a8)
* Anoop Elias [06aa46eda73d02b38c924c01806aac7eece5d58b](https://github.com/bytecodealliance/wasm-tools/commit/06aa46eda73d02b38c924c01806aac7eece5d58b)
* Arpad Borsos [4fce4f1dcd2ec23fb837a5a8b78989ccca03b05e](https://github.com/bytecodealliance/wasm-tools/commit/4fce4f1dcd2ec23fb837a5a8b78989ccca03b05e)
* Bailey Hayes [3fe7b1d597b409f4d807464d97f2815f2ceab10c](https://github.com/bytecodealliance/wasm-tools/commit/3fe7b1d597b409f4d807464d97f2815f2ceab10c)
* Ben Kimock [43860b840c58a5cfa4bf95ee669dd9326edf55b5](https://github.com/bytecodealliance/wasm-tools/commit/43860b840c58a5cfa4bf95ee669dd9326edf55b5)
* Benjamin Bouvier [72284dcf7cfe6dc37774a8bae651c382435ed8de](https://github.com/bytecodealliance/wasm-tools/commit/72284dcf7cfe6dc37774a8bae651c382435ed8de) [187fbf2c7d8e90d753bd39a472d449dfe0acd4cc](https://github.com/bytecodealliance/wasm-tools/commit/187fbf2c7d8e90d753bd39a472d449dfe0acd4cc)
* Cameron Bytheway [d98c4bc1356304f8ab3629eecad910ce5c01f45f](https://github.com/bytecodealliance/wasm-tools/commit/d98c4bc1356304f8ab3629eecad910ce5c01f45f) [0fe604a0f72fcb37876dc789014109a0d36a5a1d](https://github.com/bytecodealliance/wasm-tools/commit/0fe604a0f72fcb37876dc789014109a0d36a5a1d)
* Christian Holler [27b0d741e95eb34deb7266d766437157dbd054c2](https://github.com/bytecodealliance/wasm-tools/commit/27b0d741e95eb34deb7266d766437157dbd054c2)
* Christof Petig [031fc6141878b3be9c37f2ea1247814fb3451640](https://github.com/bytecodealliance/wasm-tools/commit/031fc6141878b3be9c37f2ea1247814fb3451640) [1791a8f139722e9f8679a2bd3d8e423e55132b22](https://github.com/bytecodealliance/wasm-tools/commit/1791a8f139722e9f8679a2bd3d8e423e55132b22) [6f59f98f7c4f94632df7c88627bbdc46db81ab02](https://github.com/bytecodealliance/wasm-tools/commit/6f59f98f7c4f94632df7c88627bbdc46db81ab02)
* Christoph Hegemann [97cad532dd3eb55933c526aff53bcda8e2f57a17](https://github.com/bytecodealliance/wasm-tools/commit/97cad532dd3eb55933c526aff53bcda8e2f57a17) [72bc318a2a9e3095f4f2aa6f80f2a0f7c1ceca88](https://github.com/bytecodealliance/wasm-tools/commit/72bc318a2a9e3095f4f2aa6f80f2a0f7c1ceca88)
* Corey Farwell [48fad3b68fb0db52fbc2cc5215edbd4761afd078](https://github.com/bytecodealliance/wasm-tools/commit/48fad3b68fb0db52fbc2cc5215edbd4761afd078)
* Daniel Bevenius [e1cca2747439ebfc020de1248240d3ddc9f1e5c5](https://github.com/bytecodealliance/wasm-tools/commit/e1cca2747439ebfc020de1248240d3ddc9f1e5c5)
* Daniel Macovei [8b9ba68a18f1bac2f31c4889cc6c462cde3f13a0](https://github.com/bytecodealliance/wasm-tools/commit/8b9ba68a18f1bac2f31c4889cc6c462cde3f13a0) [e7b78724dca8f59a4f679a0b715f90a5625a3d8f](https://github.com/bytecodealliance/wasm-tools/commit/e7b78724dca8f59a4f679a0b715f90a5625a3d8f) [30c7a0948c62df8b2fd578b16853120f457c0bb6](https://github.com/bytecodealliance/wasm-tools/commit/30c7a0948c62df8b2fd578b16853120f457c0bb6)
* Daniel Salvadori [fc2a190720f3a8d66ff9fc2af17bade898599da5](https://github.com/bytecodealliance/wasm-tools/commit/fc2a190720f3a8d66ff9fc2af17bade898599da5)
* Danielle Maywood [6e27a4371be13184e827c86c60e916ceb7ab2557](https://github.com/bytecodealliance/wasm-tools/commit/6e27a4371be13184e827c86c60e916ceb7ab2557)
* David Haynes [a6f7345c4b82256a0a4658cdca7caa204c4a3e4b](https://github.com/bytecodealliance/wasm-tools/commit/a6f7345c4b82256a0a4658cdca7caa204c4a3e4b)
* David Yamnitsky [55f65f7c7383a9cac2e8e92d8518822e20d9e9af](https://github.com/bytecodealliance/wasm-tools/commit/55f65f7c7383a9cac2e8e92d8518822e20d9e9af)
* Dylan Thacker-Smith [73256499fe1235dcef7549b59d6cbae46a948afd](https://github.com/bytecodealliance/wasm-tools/commit/73256499fe1235dcef7549b59d6cbae46a948afd)
* Gabor Greif [168a76b756d01603366850dede813a1442f7f0b9](https://github.com/bytecodealliance/wasm-tools/commit/168a76b756d01603366850dede813a1442f7f0b9)
* George Kulakowski [4dff58c1fe7ebb77dcd6bc033cc7b720b4f2cd5c](https://github.com/bytecodealliance/wasm-tools/commit/4dff58c1fe7ebb77dcd6bc033cc7b720b4f2cd5c) [9cab263120fba742e743a842c205091b466e524f](https://github.com/bytecodealliance/wasm-tools/commit/9cab263120fba742e743a842c205091b466e524f) [a2788141deb3d0e6dd92fe4549033980fa8b4e48](https://github.com/bytecodealliance/wasm-tools/commit/a2788141deb3d0e6dd92fe4549033980fa8b4e48)
* Iceber Gu [b99ff3e873c87afa0d6d673ca1ffe336f045cd4d](https://github.com/bytecodealliance/wasm-tools/commit/b99ff3e873c87afa0d6d673ca1ffe336f045cd4d)
* Ivan Enderlin [d31367323ab097c18e2047f01a99c709731f4934](https://github.com/bytecodealliance/wasm-tools/commit/d31367323ab097c18e2047f01a99c709731f4934) [64b5091a27a4be17883f44437f06f621c80367ab](https://github.com/bytecodealliance/wasm-tools/commit/64b5091a27a4be17883f44437f06f621c80367ab)
* Jakob Stoklund Olesen [48438b86e8756640f9bbd4e9b7707491d7a9beef](https://github.com/bytecodealliance/wasm-tools/commit/48438b86e8756640f9bbd4e9b7707491d7a9beef)
* Jakub Konka [760634c4bce202ccd5ecbfad8c08e57129a00d51](https://github.com/bytecodealliance/wasm-tools/commit/760634c4bce202ccd5ecbfad8c08e57129a00d51) [a612654ddebe763f99286a802322240f74bff47e](https://github.com/bytecodealliance/wasm-tools/commit/a612654ddebe763f99286a802322240f74bff47e)
* James [Undefined] [8dc75f042f92cd8d28cda1054a213c4b0d8e9b75](https://github.com/bytecodealliance/wasm-tools/commit/8dc75f042f92cd8d28cda1054a213c4b0d8e9b75) [19cc2ab3c09d9759429d8a8e16f501585373d4ab](https://github.com/bytecodealliance/wasm-tools/commit/19cc2ab3c09d9759429d8a8e16f501585373d4ab)
* Johan Andersson [0d717c4bee87ff2b6d12c83d35d8f7c1db5475fd](https://github.com/bytecodealliance/wasm-tools/commit/0d717c4bee87ff2b6d12c83d35d8f7c1db5475fd)
* Josh Stone [4fdfc3869b988bfcdb7660f61b38b685f8389d94](https://github.com/bytecodealliance/wasm-tools/commit/4fdfc3869b988bfcdb7660f61b38b685f8389d94)
* Klim Tsoutsman [27c1817b0e6b267e0fc5fd79c157ee91cdfac689](https://github.com/bytecodealliance/wasm-tools/commit/27c1817b0e6b267e0fc5fd79c157ee91cdfac689)
* Koute [d550bd5491f9433bc42601f03a632ade53805b62](https://github.com/bytecodealliance/wasm-tools/commit/d550bd5491f9433bc42601f03a632ade53805b62)
* Kunal Purohit [957be4a4239ab753bc63c33fcc734823f92edb7d](https://github.com/bytecodealliance/wasm-tools/commit/957be4a4239ab753bc63c33fcc734823f92edb7d)
* Léo Gaspard [dc1480da70db31ccca31c44569c2ce1ea7f86d5b](https://github.com/bytecodealliance/wasm-tools/commit/dc1480da70db31ccca31c44569c2ce1ea7f86d5b) [24d48083cea015bd62887f9b141caf7a18e5505e](https://github.com/bytecodealliance/wasm-tools/commit/24d48083cea015bd62887f9b141caf7a18e5505e)
* Mark McCaskey [fc5dccc4ca0c93c488990509b2ca6d2fb042d796](https://github.com/bytecodealliance/wasm-tools/commit/fc5dccc4ca0c93c488990509b2ca6d2fb042d796) [ac6df05378622c198b765565016fc8b068680b74](https://github.com/bytecodealliance/wasm-tools/commit/ac6df05378622c198b765565016fc8b068680b74)
* Martin Rueckl [aaf941dbda39029b036f3a9390fe4d3525105774](https://github.com/bytecodealliance/wasm-tools/commit/aaf941dbda39029b036f3a9390fe4d3525105774)
* Matheus Cardoso [9502bf5b44f4fe1ebf10355f13ebbb89f8dc1a28](https://github.com/bytecodealliance/wasm-tools/commit/9502bf5b44f4fe1ebf10355f13ebbb89f8dc1a28) [6e4e066d16dd820ef1374ec4f9c321804f60a99e](https://github.com/bytecodealliance/wasm-tools/commit/6e4e066d16dd820ef1374ec4f9c321804f60a99e)
* Michelle Dhanani [0263c2f2e0a33fe97569535aa2784b7989cb1dd0](https://github.com/bytecodealliance/wasm-tools/commit/0263c2f2e0a33fe97569535aa2784b7989cb1dd0) [e18dc3d0890b0b3a19597ec76593ec70b27e9263](https://github.com/bytecodealliance/wasm-tools/commit/e18dc3d0890b0b3a19597ec76593ec70b27e9263)
* Noah Jelich [9affab2262766b10ce0a2480677a5ac6c279faf5](https://github.com/bytecodealliance/wasm-tools/commit/9affab2262766b10ce0a2480677a5ac6c279faf5) [a2761cc61df2b51518fdf73df4cc3cb6ab97efcf](https://github.com/bytecodealliance/wasm-tools/commit/a2761cc61df2b51518fdf73df4cc3cb6ab97efcf)
* Oliver Horn [3d5005f501d78c544ac9106064a5ecd575b1f67e](https://github.com/bytecodealliance/wasm-tools/commit/3d5005f501d78c544ac9106064a5ecd575b1f67e)
* Olivier Lemasle [09436a16f128fce0c1de812d0b46f8975253fbfe](https://github.com/bytecodealliance/wasm-tools/commit/09436a16f128fce0c1de812d0b46f8975253fbfe)
* Omer Sadika [113d753d96f4ef2d6f7acfc14d4337ba4afa3ae5](https://github.com/bytecodealliance/wasm-tools/commit/113d753d96f4ef2d6f7acfc14d4337ba4afa3ae5) [ccfb6055ea760537c5c9ae3761ccb6a54ae50181](https://github.com/bytecodealliance/wasm-tools/commit/ccfb6055ea760537c5c9ae3761ccb6a54ae50181)
* Onuwa Nnachi Isaac [4c2786449b2483d85b88a126ba18a4195d1561bd](https://github.com/bytecodealliance/wasm-tools/commit/4c2786449b2483d85b88a126ba18a4195d1561bd)
* Radu M [3d52fe606c34513a66d0f4403dfb6098db0d1c8d](https://github.com/bytecodealliance/wasm-tools/commit/3d52fe606c34513a66d0f4403dfb6098db0d1c8d)
* Rajat Jindal [2ebe0d8755908eb0ba62eb1103fd3e80abcb1eb2](https://github.com/bytecodealliance/wasm-tools/commit/2ebe0d8755908eb0ba62eb1103fd3e80abcb1eb2)
* Robin Brown [cc5f620f74c577a5ca2710d14912e9f42d2fa8bd](https://github.com/bytecodealliance/wasm-tools/commit/cc5f620f74c577a5ca2710d14912e9f42d2fa8bd) [d299a46c379f545411da940c3c52654ee1c70647](https://github.com/bytecodealliance/wasm-tools/commit/d299a46c379f545411da940c3c52654ee1c70647) [e0dd9df85fa482585c76acecb52193b0405689d9](https://github.com/bytecodealliance/wasm-tools/commit/e0dd9df85fa482585c76acecb52193b0405689d9)
* Roman Volosatovs [3af2562b91fbe5ea225f193b3dc2747142dce18e](https://github.com/bytecodealliance/wasm-tools/commit/3af2562b91fbe5ea225f193b3dc2747142dce18e)
* Sam Sartor [cad6975c5f163107534e6ec3877b689727c1463b](https://github.com/bytecodealliance/wasm-tools/commit/cad6975c5f163107534e6ec3877b689727c1463b)
* Saúl Cabrera [dd3c6b97e8642ad4851c301b5b8803e47fc6119d](https://github.com/bytecodealliance/wasm-tools/commit/dd3c6b97e8642ad4851c301b5b8803e47fc6119d) [eab71f86b66778acbeaa03cb081c6fe78aaa1144](https://github.com/bytecodealliance/wasm-tools/commit/eab71f86b66778acbeaa03cb081c6fe78aaa1144) [f8200cf02a1e69dee33335bbd1dada84d31eb783](https://github.com/bytecodealliance/wasm-tools/commit/f8200cf02a1e69dee33335bbd1dada84d31eb783)
* Sergei Shulepov [9339e0ff39ce9bc7b6c65041f3bfd12c77226114](https://github.com/bytecodealliance/wasm-tools/commit/9339e0ff39ce9bc7b6c65041f3bfd12c77226114)
* Stephan Renatus [289013a86e0c96513cf3bfc82a25d32fc1d6bad1](https://github.com/bytecodealliance/wasm-tools/commit/289013a86e0c96513cf3bfc82a25d32fc1d6bad1)
* Syrus Akbary [371445d016de673ee17e29fab0f988f584f0c211](https://github.com/bytecodealliance/wasm-tools/commit/371445d016de673ee17e29fab0f988f584f0c211) [7da664580a5e8a74569fd9a75ce9a59021fc96ac](https://github.com/bytecodealliance/wasm-tools/commit/7da664580a5e8a74569fd9a75ce9a59021fc96ac)
* Sérgio da Gama [755abb691d9d401dad542b8f238d19e6f3624e16](https://github.com/bytecodealliance/wasm-tools/commit/755abb691d9d401dad542b8f238d19e6f3624e16) [56a29632f09cd544ead6351d55045b9fbbda51b4](https://github.com/bytecodealliance/wasm-tools/commit/56a29632f09cd544ead6351d55045b9fbbda51b4)
* Takanori Ishibashi [4f83e78d58f79fc71265510165a22bf86dee619a](https://github.com/bytecodealliance/wasm-tools/commit/4f83e78d58f79fc71265510165a22bf86dee619a)
* Takumasa Sakao [03d21b8d9f0dd29441c5de4d6a2fc1505a9fd0d5](https://github.com/bytecodealliance/wasm-tools/commit/03d21b8d9f0dd29441c5de4d6a2fc1505a9fd0d5)
* Thaumy [f138a5b022cbb74f32eb09e0fbe9d5b5b57760b3](https://github.com/bytecodealliance/wasm-tools/commit/f138a5b022cbb74f32eb09e0fbe9d5b5b57760b3)
* Till Schneidereit [26e2e84a5e823969c9751cf71918aa74443e7443](https://github.com/bytecodealliance/wasm-tools/commit/26e2e84a5e823969c9751cf71918aa74443e7443) [8ddb0c89238a9f8c82fca555951f77d11e5abe03](https://github.com/bytecodealliance/wasm-tools/commit/8ddb0c89238a9f8c82fca555951f77d11e5abe03) [bd36dd1a02fbfb55f54a38d65c9669a489b8f761](https://github.com/bytecodealliance/wasm-tools/commit/bd36dd1a02fbfb55f54a38d65c9669a489b8f761)
* Victor Adossi [30cc3aec517fff3670556bc7f28dee27c0434af8](https://github.com/bytecodealliance/wasm-tools/commit/30cc3aec517fff3670556bc7f28dee27c0434af8) [67fb7b002cb6c01ed647766afc4b50c422faafb9](https://github.com/bytecodealliance/wasm-tools/commit/67fb7b002cb6c01ed647766afc4b50c422faafb9)
* Viktor Charypar [90e62051c0ffcad0301c1375fb9b482eef4c0ba4](https://github.com/bytecodealliance/wasm-tools/commit/90e62051c0ffcad0301c1375fb9b482eef4c0ba4)
* Vinay Chandra [035389ef3a5755119153ae7302dcc7112d40eb78](https://github.com/bytecodealliance/wasm-tools/commit/035389ef3a5755119153ae7302dcc7112d40eb78)
* XAMPPRocky [3be1da96371086bbd1794e3eb20c0e7550c114e6](https://github.com/bytecodealliance/wasm-tools/commit/3be1da96371086bbd1794e3eb20c0e7550c114e6) [299b0854c0ef9a568faf2c0f52e5cffd6d3dc003](https://github.com/bytecodealliance/wasm-tools/commit/299b0854c0ef9a568faf2c0f52e5cffd6d3dc003)
* __SyS__ [c5cda464edba41a4e1c43c38dc4a68ea93a9730c](https://github.com/bytecodealliance/wasm-tools/commit/c5cda464edba41a4e1c43c38dc4a68ea93a9730c)
* cdetrio [08cc5ef042046a5764c0a78e226fdb69bd9e4b5b](https://github.com/bytecodealliance/wasm-tools/commit/08cc5ef042046a5764c0a78e226fdb69bd9e4b5b)
* iqbigbang [65ec81c5d4e5070c32a594134ff3be490293c51f](https://github.com/bytecodealliance/wasm-tools/commit/65ec81c5d4e5070c32a594134ff3be490293c51f)
* xortoast [ac2c8e31560a4981264e5f6f65eda19854f88315](https://github.com/bytecodealliance/wasm-tools/commit/ac2c8e31560a4981264e5f6f65eda19854f88315) [3864f08213a191fddc143f00608fb4eabd4439fc](https://github.com/bytecodealliance/wasm-tools/commit/3864f08213a191fddc143f00608fb4eabd4439fc) [10949db43a3d39d1d9e676d8b8f7b5f8dd4e3793](https://github.com/bytecodealliance/wasm-tools/commit/10949db43a3d39d1d9e676d8b8f7b5f8dd4e3793)
* zu1k [8d14fa54940b4276d992fae8c9ac8e01b8126eff](https://github.com/bytecodealliance/wasm-tools/commit/8d14fa54940b4276d992fae8c9ac8e01b8126eff)
* Ömer Sinan Ağacan [e32627b60fd5f336f66a10348fbf88a319c10c26](https://github.com/bytecodealliance/wasm-tools/commit/e32627b60fd5f336f66a10348fbf88a319c10c26)
* 韩朴宇 [9340ed2466a50b4dbc580b13ba18a417dee91433](https://github.com/bytecodealliance/wasm-tools/commit/9340ed2466a50b4dbc580b13ba18a417dee91433)

**Comment by jbms** [comment]
I approve this license change.

**Comment by fibonacci1729** [comment]
I approve this license change.

**Comment by MendyBerger** [comment]
I approve this license change.

**Comment by imikushin** [comment]
I approve this license change.

**Comment by peterhuene** [comment]
I approve this license change.

**Comment by yurydelendik** [comment]
I approve this license change

**Comment by tschneidereit** [comment]
For external visibility: this kind of license change requires board approval, as documented in section 4 of the Bytecode Alliance's [IP policy](https://bytecodealliance.org/assets/ip-policy.pdf). The board discussed this back in March and granted the exception.

**Comment by bakkot** [comment]
I approve this license change.

**Comment by dhil** [comment]
I approve this license change.

**Comment by dicej** [comment]
I approve this license change.

**Comment by Robbepop** [comment]
I approve this license change.

**Comment by nagisa** [comment]
I approve this license change

**Comment by elliottt** [comment]
I approve this license change

**Comment by ydnar** [comment]
I approve this licensing change.

**Comment by pchickey** [comment]
I approve this license change.

**Comment by juntyr** [comment]
I approve this license change.

**Comment by fitzgen** [comment]
I approve this license change

**Comment by cfallin** [comment]
I approve this licensing change.

**Comment by sunfishcode** [comment]
I approve this license change.

**Comment by bvisness** [comment]
I approve this license change.

**Comment by azaslavsky** [comment]
I approve this license change.

**Comment by YaronWittenstein** [comment]
I approve this license change.

**Comment by yoshuawuyts** [comment]
I approve this license change.

**Comment by lann** [comment]
I approve this license change.

**Comment by eqrion** [comment]
I approve this license change.

**Comment by abrown** [comment]
I approve this license change

**Comment by Liamolucko** [comment]
I approve this license change.

**Comment by andylizi** [comment]
I approve this license change.

**Comment by kateinoigakukun** [comment]
I approve this license change.

**Comment by maekawatoshiki** [comment]
I approve this license change.

**Comment by lars-t-hansen** [comment]
I approve this license change.

**Comment by rylev** [comment]
I approve this license change.

**Comment by adambratschikaye** [comment]
I approve this license change

**Comment by takikawa** [comment]
I approve this license change.

**Comment by silesmo** [comment]
I approve this license change.

**Comment by Mossaka** [comment]
I approve this license change

**Comment by yowl** [comment]
I approve this license change.

**Comment by Jacarte** [comment]
I approve this license change

**Comment by alexcrichton** [comment]
I reached out to darinmorrison over email and received confirmation from them.

**Comment by alexcrichton** [comment]
@syrusakbary, I swapped you in for @nlewycky: over email, @nlewycky indicated that you'd need to sign off on behalf of Wasmer as the copyright holder, since these contributions were done while they worked at Wasmer.

**Comment by mainrs** [comment]
I approve this license change.

**Comment by bholley** [comment]
I approve this change for the work contributed by Mozilla staff (Michael and Julian).

**Comment by tschneidereit** [comment]
Alex and I discussed how to handle the remaining contributions for which we don't have sign-off. Instead of waiting and hoping that eventually we might get that sign-off, we assessed the contributions in more detail than before.

The result is that there's only a single commit we still don't have sign-off for where we'd actually need it: the C/C++ bindings [contributed](https://github.com/bytecodealliance/wasm-tools/commit/77f171841d1a47e33e25cf49369e82c3d1b18e07) by @TheGreatRambler.

This is a meaningful contribution, and we'd very clearly need sign-off to change the license of this code. So instead, we'll keep this code under the current license: it's not something of interest for use by Rust anyway.

The remaining commit we don't have sign-off for were contributed by @nlewycky as a Wasmer employee. A detailed look at these contributions shows that most of them are trivial in the copyright sense (not necessarily in the sense that making these contributions was trivial! Contributions often require careful debugging or investigation work, only to result in small changes that don't clear the creativeness bar copyright requires.) Those parts of the contributions that could even remotely be candidates for copyright happen to not be part of the code base as it is today.

Here's a list of the 11 commits in question, with explanations of why we don't need sign-off for them:
 06e5bcc: trivial change
 d387165: trivial change
 59f1881: trivial change
 73dff98: pretty trivial, but also removed in its entirety
 8bdcd81: pretty trivial, but also removed in its entirety
 96a03c1: pretty trivial, but also removed in its entirety
 dfc6f3d: trivial change
 32b68b8: trivial change, with anything marginally interesting removed in its entirety
 319a2e2: removed in its entirety
 cb3bb10: trivial change
 ea17086: trivial change
